contracts/upgradeable_contracts/amb_erc677_to_erc677/BasicAMBErc677ToErc677.sol

```diff
@@ -0,0 +1,188 @@
+pragma solidity 0.4.24;
+
+import "../../interfaces/IAMB.sol";
+import "../Ownable.sol";
+import "openzeppelin-solidity/contracts/AddressUtils.sol";
+import "../Initializable.sol";
+import "../ERC677Bridge.sol";
+import "../BaseOverdrawManagement.sol";
+import "../ReentrancyGuard.sol";
+import "../Upgradeable.sol";
+import "../Claimable.sol";
+import "../VersionableBridge.sol";
+
+contract BasicAMBErc677ToErc677 is
+    Initializable,
+    Ownable,
+    ReentrancyGuard,
+    Upgradeable,
+    Claimable,
+    VersionableBridge,
```

Comment:
Thanks for the question. Actually I think it would be better to have its own separate versioning. Changed here https://github.com/poanetwork/poa-bridge-contracts/pull/273/commits/398350b304cc0388614588af966afbc07dc18b8d